devel/gdb/Portfile

```diff
@@ -33,9 +32,9 @@ supported_archs x86_64 i386
 
 master_sites    gnu
 
-checksums       rmd160  82c6d7d2c96b4716f9f338b116122b7395b07e90 \
-                sha256  26ce655216cd03f4611518a7a1c31d80ec8e884c16715e9ba8b436822e51434b \
-                size    38130999
+checksums           rmd160  fd9cc7af22ed0700f5de7a2bc21e79b5353e6385 \
+                    sha256  fcda54d4f35bc53fb24b50009a71ca98410d71ff2620942e3c829a7f5d614252 \
+                    size    39255376
```

Comment:
You've changed the indentation of the checksums lines; please fix it to be consistent with the rest of the Portfile (i.e. the way it was before.)

Reply:
I reverted it to how it was before. I guess `port -d checksum` gave a couple more spaces than was necessary.